packages/manager/src/components/AreaChart/AreaChart.tsx

```diff
@@ -139,11 +150,13 @@ export const AreaChart = (props: AreaChartProps) => {
     ariaLabel,
     connectNulls,
     data,
+    dotRadius = 3,
```

Comment:
I think we agreed upon 2?

Reply:
that's the default value for everyone, and as for our case I've discussed with Kiran & seem 1.5 is fine for us, that we will pass as prop.

Reply:
3 is actually looks cluttered for DBaaS kind of data where we have multiple series with too granular data of 1 min.